packages/camera/camera/ios/Classes/CameraPlugin.m

```diff
@@ -1030,6 +1031,16 @@ - (void)applyFocusMode:(FocusMode)focusMode onDevice:(AVCaptureDevice *)captureD
   [captureDevice unlockForConfiguration];
 }
 
+- (void)pausePreviewWithResult:(FlutterResult)result {
+  _isPreviewPaused = true;
+  result(nil);
+}
+
+- (void)resumePreviewWithResult:(FlutterResult)result {
+  _isPreviewPaused = false;
+  result(nil);
+}
```

Comment:
It would be good to add unit-tests here to verify that `result(nil)` is indeed called.